index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css"/>
+    <title>Pete Heyes - Developer Chap</title>
```

Comment:
Title placement?

Reply:
not sure what this means. please can you give more detail. sorry!

Reply:
Keep all links together